build-bin/deploy

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
probably the license headers in these files are obnoxious.. in this directory it is only small logic, so maybe we should kill the headers (unlike docker/ which has more interesting stuff)